test/fixtures/articles.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
is this file necessary?

Reply:
The file articles.yml was generated with the model for the articles. It's only necessary if tests with the articles are required.